scripts/Interface/reduction_gui/instruments/instrument_factory.py

```diff
@@ -1,3 +1,4 @@
+#pylint: disable = W0403
```

Comment:
Absolutely - in my .py file I did that, but did not want to do it here, since this one is not mine and I've only added another line following the existing pattern. Do I have a green light to change the imports in this file?